### DISCUSSION TITLE
Where to find MongoDB connection string

### DISCUSSION THREAD

**Discussion by dvorak07patrik** [br-g/openf1#38] [Q&A] [answered]
Hello, I encountered an issue while attempting to access the full database. The connection string provided in the README.md file at openf1/query_api/ doesn't seem to work for me. Could you assist me with this?

**Comment by br-g** [comment, marked as answer]
Hello,

This connection string is now inactive. Sorry for that!
We will introduce a request rate limit per user via the API shortly, to reduce cost. Direct MongoDB access has been deactivated due to its inability to restrict user request rates
